Description: No longer use rt.jar which is gone since Java 9
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: no
--- a/make/build.xml
+++ b/make/build.xml
@@ -729,7 +729,6 @@
            encoding="UTF-8"
            source="${target.sourcelevel}" 
            target="${target.targetlevel}" 
-           bootclasspath="${target.rt.jar}"
            debug="${javacdebug}" debuglevel="${javacdebuglevel}">
       <src path="${src.java}" />
       <src path="${src.generated.java}" />
@@ -757,7 +756,6 @@
            encoding="UTF-8"
            source="${target.sourcelevel}"
            target="${target.targetlevel}" 
-           bootclasspath="${host.rt.jar}"
            debug="${javacdebug}" debuglevel="${javacdebuglevel}">
       <src path="${src.jcpp}" />
       <src path="${src.java}" />
@@ -914,7 +912,6 @@
            encoding="UTF-8"
            source="${target.sourcelevel}" 
            target="${target.targetlevel}" 
-           bootclasspath="${target.rt.jar}"
            debug="${javacdebug}" debuglevel="${javacdebuglevel}">
       <src path="${src.java}" />
       <src path="${src.generated.java}" />
@@ -974,7 +971,6 @@
            encoding="UTF-8"
            source="${target.sourcelevel}" 
            target="${target.targetlevel}" 
-           bootclasspath="${target.rt.jar}"
            debug="${javacdebug}" debuglevel="${javacdebuglevel}">
       <src path="${src.java}" />
       <classpath refid="android.classpath" />
--- a/make/jogamp-env.xml
+++ b/make/jogamp-env.xml
@@ -183,13 +183,6 @@
     <echo message="target.sourcelevel         ${target.sourcelevel}"/>
     <echo message="target.targetlevel         ${target.targetlevel}"/>
     <echo message="target.rt.jar              ${target.rt.jar}"/>
-    <fail message="No target.rt.jar specified not found.">
-      <condition>
-        <not>
-          <isset property="target.rt.jar"/>
-        </not>
-      </condition>
-    </fail>
 
     <property name="host.sourcelevel"         value="${ant.java.version}" />      <!-- default values for compile time components -->
     <property name="host.targetlevel"         value="${ant.java.version}" />      <!-- default values for compile time components -->
@@ -206,13 +199,6 @@
     <echo message="host.sourcelevel           ${host.sourcelevel}"/>
     <echo message="host.targetlevel           ${host.targetlevel}"/>
     <echo message="host.rt.jar                ${host.rt.jar}"/>
-    <fail message="No host.rt.jar specified not found.">
-      <condition>
-        <not>
-          <isset property="host.rt.jar"/>
-        </not>
-      </condition>
-    </fail>
 
     <fail message="Unsupported Target Java version (sourcelevel): ${target.sourcelevel}. Make sure that the version of the Java compiler is 1.6 (6.0) or greater.">
         <condition>
